base/loading.jl

```diff
@@ -88,7 +88,7 @@ macro return_if_file(path)
 end
 
 function find_package(name::String)
-    endswith(name, ".jl") && (name = chop(name, 0, 3))
+    endswith(name, ".jl") && (name = chop(name, head = 0, tail = 3))
```

Comment:
Here you can drop the `head = 0` since that's the default. It was only needed for the positional form because the `head` argument is first.

Reply:
^ bump